Improve Landing Page Design

GeMini-220 opened the pull request.

GeMini-220 committed:
Footer & Font fixes

GeMini-220 committed:
Update banner design

dependabot[bot] committed:
Dependencies(deps-dev): Bump eslint-plugin-vue from 7.10.0 to 7.11.0 (#302)

Bumps [eslint-plugin-vue](https://github.com/vuejs/eslint-plugin-vue) from 7.10.0 to 7.11.0.
- [Release notes](https://github.com/vuejs/eslint-plugin-vue/releases)
- [Commits](https://github.com/vuejs/eslint-plugin-vue/compare/v7.10.0...v7.11.0)

---
updated-dependencies:
- dependency-name: eslint-plugin-vue
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

GeMini-220 committed:
Footer adaptive layout

GeMini-220 committed:
Add News & Community Components

GeMini-220 committed:
display fixes

GeMini-220 committed:
responsive contents

GeMini-220 committed:
add entry animations

GeMini-220 committed:
interactive designs

Josh-Cena reviewed (changes requested):
- [ ] I still like the dark footer so that there's a visual contrast from the main content, just like on [our website](https://computerization.io). WDYT?
- [x] The carousel is broken, since the bottom half of the image is missing.
- [ ] The "news" section looks great, but consider darkening the color since the yellow is too bright and saturated. (Also, out of curiosity: what kind of news should we put here?)
- [x] When hovering above a news item, the item turns white. I believe this is not the intended behavior.

Josh-Cena commented on src/App.vue:
```diff
@@ -1,13 +1,13 @@
 <template>
   <v-app>
-    <Appbar @toggle-sidebar="toggleSidebar" />
+    <Appbar style="z-index: 999" @toggle-sidebar="toggleSidebar" />
```
Don't use inline style—move everything to the `style` tag.

GeMini-220 replied:
Was trying to overwrite some of the Vuetify stylesheets XD
cant find a better way.. for now. I believe I tried to use "!important" but sometimes it doesn't work.
Worried if I change the configuration files directly it might be overwritten when Vuetify updates..

GeMini-220 commented:
> * [ ]  I still like the dark footer so that there's a visual contrast from the main content, just like on [our website](https://computerization.io). WDYT?
> * [ ]  The carousel is broken, since the bottom half of the image is missing.
> * [ ]  The "news" section looks great, but consider darkening the color since the yellow is too bright and saturated. (Also, out of curiosity: what kind of news should we put here?)
> * [ ]  When hovering above a news item, the item turns white. I believe this is not the intended behavior.

 @Josh-Cena 
Most functions looks right on Google Chrome, but its quite strange that they doesn't work in Microsoft Edge.... might be caused by some CSS version issues... I'll try to fix this after the exam
(+ perhaps I'll also remodify the design, again..... or maybe I could add a "dark theme" ..? like most of the softs do... This 
might be able meet the taste of more people)

Josh-Cena commented:
I'm using Chrome FYI. And this is how it looks:

![image](https://user-images.githubusercontent.com/55398995/121839150-188b0e00-cd0c-11eb-9b54-71593e2e0162.png)

![image](https://user-images.githubusercontent.com/55398995/121839160-1e80ef00-cd0c-11eb-9a93-61a5259a0523.png)

dependabot[bot] committed:
Dependencies(deps-dev): Bump eslint-plugin-vue from 7.11.0 to 7.11.1 (#304)

dependabot[bot] committed:
Dependencies(deps-dev): Bump typescript from 4.3.2 to 4.3.3 (#310)

dependabot[bot] committed:
Dependencies(deps-dev): Bump @typescript-eslint/parser (#306)

Bumps [@typescript-eslint/parser](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/parser) from 4.26.1 to 4.27.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/parser/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v4.27.0/packages/parser)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/parser"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Dependencies(deps): Bump vuetify from 2.5.3 to 2.5.4 (#308)

Bumps [vuetify](https://github.com/vuetifyjs/vuetify/tree/HEAD/packages/vuetify) from 2.5.3 to 2.5.4.
- [Release notes](https://github.com/vuetifyjs/vuetify/releases)
- [Commits](https://github.com/vuetifyjs/vuetify/commits/v2.5.4/packages/vuetify)

---
updated-dependencies:
- dependency-name: vuetify
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump postcss from 7.0.35 to 7.0.36 (#311)

Bumps [postcss](https://github.com/postcss/postcss) from 7.0.35 to 7.0.36.
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/7.0.35...7.0.36)

---
updated-dependencies:
- dependency-name: postcss
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Dependencies(deps-dev): Bump sass from 1.34.1 to 1.35.1 (#309)

Bumps [sass](https://github.com/sass/dart-sass) from 1.34.1 to 1.35.1.
- [Release notes](https://github.com/sass/dart-sass/releases)
- [Changelog](https://github.com/sass/dart-sass/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sass/dart-sass/compare/1.34.1...1.35.1)

---
updated-dependencies:
- dependency-name: sass
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Dependencies(deps-dev): Bump @typescript-eslint/eslint-plugin (#307)

Bumps [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/eslint-plugin) from 4.26.1 to 4.27.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v4.27.0/packages/eslint-plugin)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/eslint-plugin"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Dependencies(deps-dev): Bump typescript from 4.3.3 to 4.3.4 (#312)

Bumps [typescript](https://github.com/Microsoft/TypeScript) from 4.3.3 to 4.3.4.
- [Release notes](https://github.com/Microsoft/TypeScript/releases)
- [Commits](https://github.com/Microsoft/TypeScript/compare/v4.3.3...v4.3.4)

---
updated-dependencies:
- dependency-name: typescript
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

GeMini-220 commented:
@Josh-Cena Does the banner looks normal now? I was not sure what actually caused the problem but those looks normal on my browsers now. I did rebased the branch. might be solved by bumping the Vuetify..?

This is how it looks now:

On Microsoft Edge
![df62433b93ee3f56626661e31c03567](https://user-images.githubusercontent.com/71819626/122631235-de9b7c80-d0fc-11eb-9923-bb06c08a621d.png)

On Google Chrome
![21f852d7e0a362c1b7ad45578c80738](https://user-images.githubusercontent.com/71819626/122631237-e8bd7b00-d0fc-11eb-964c-25fe26c0e46d.png)

GeMini-220 committed:
interactive design fixes

Josh-Cena commented:
Yea, so first, you should add a right margin so that the carousel edge doesn't touch the page side:

![image](https://user-images.githubusercontent.com/55398995/122631384-04755100-d0fe-11eb-911f-77e01f08bf2a.png)

Then, I don't really know what's going on, but seems the `height: 200px` is cropping the image rather than scaling it:

![image](https://user-images.githubusercontent.com/55398995/122631432-633aca80-d0fe-11eb-81ff-38b1ae517f16.png)

The nuance is probably your browser window is not at full size: mine has width of 1440px.

GeMini-220 committed:
add app bar switch

GeMini-220 committed:
fix banner

GeMini-220 committed:
install less.js

GeMini-220 committed:
less compatability fix

GeMini-220 committed:
reorganize and conform light-theme colors

GeMini-220 committed:
seperate public theme.less file

GeMini-220 committed:
landing page switch to responsive colors

GeMini-220 committed:
add dark-theme colors

GeMini-220 committed:
fixes

GeMini-220 committed:
style fixes

GeMini-220 committed:
switch alignment fix

GeMini-220 committed:
data transmission fixes

GeMini-220 committed:
data transmission fixes

GeMini-220 committed:
update dependencies

GeMini-220 committed:
data transmission fixes

GeMini-220 committed:
dependencies fixes

GeMini-220 committed:
typescript fixes

GeMini-220 committed:
themes colors update

qwerzl commented:
Closing as we're switching to another branch rewritten from the ground up :-)
